package.json

```diff
@@ -29,6 +29,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-fast-marquee": "^1.6.0",
+    "react-icons": "^5.2.1",
```

Comment:
You don't need an icon library for this one. It is much simpler to follow the approach of adding the SVG path in the `app/components/ui/icons/icons.ts` file and then add it on the footer through the `app/components/footer/index.tsx` with the `<Icon />` component and follow the same structure of the other links.